transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/ExportAsyncJobService.groovy

```diff
@@ -160,6 +155,11 @@ class ExportAsyncJobService extends AbstractDataResourceService {
         new JobKey(jobId.toString(), 'DataExport')
     }
 
+    private static escapeInvalidFileNameChars(String fileName) {
+        // escape characters that are not on the file name characters whitelist
+        fileName?.replaceAll("[^a-zA-Z0-9_\\.\\-\\s]", "_")
```

Comment:
This quite restrictive, would also replace characters like ë, ø, ń with _. It could make the job names harder to read and increases the probability is job name collisions (we anyway need to adapt the [ExcludeValuesValidatorDirective](https://github.com/thehyve/glowing-bear/blob/dev/src/app/validators/exclude-values-validator.directive.ts#L18) in Glowing Bear to prevent duplicate job names).

An alterative could be to just replace the [reserved characters](https://en.wikipedia.org/wiki/Filename#Reserved_characters_and_words) (except `.` and ` `).

Reply:
@gijskant I updated the replacing rule and addressed the related Glowing Bear change in [TMT-839](https://jira.thehyve.nl/browse/TMT-839)